rct229/rulesets/ashrae9012019/section22/section22rule35.py

```diff
@@ -0,0 +1,29 @@
+from rct229.rule_engine.partial_rule_definition import PartialRuleDefinition
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.rulesets.ashrae9012019.ruleset_functions.check_purchased_chw_hhw_status_dict import (
+    check_purchased_chw_hhw_status_dict,
+)
+
+
+class Section22Rule35(PartialRuleDefinition):
+    """Rule 35 of ASHRAE 90.1-2019 Appendix G Section 22 (Chilled water loop)"""
+
+    def __init__(self):
+        super(Section22Rule35, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            id="22-35",
+            description="Baseline systems served by purchased chilled water shall not be modeled with"
+            " chilled water reset",
+            ruleset_section_title="HVAC - Chiller",
+            standard_section="Section G3.1.3.10 Chilled-Water Pumps (Systems 7, 8, 11, 12, and 13)",
+            is_primary_rule=False,
+            rmr_context="ruleset_model_instances/0",
+            manual_check_required_msg="Manual Check Required - Baseline is modeled with purchased chilled water.  Make sure "
+            "baseline systems served by purchased chilled water are not modeled with chilled water reset.",
+            not_applicable_msg="Rule 22-35 Not Applicable - the baseline is not modeled with Purchased Chilled Water",
+        )
+
+    def applicability_check(self, context, calc_vals, data):
+        rmi_b = context.baseline
```

Comment:
Under the PartialRuleDefinition, the level is still at the top level instead of rule model instance level.

```python
class Section22Rule35(RuleDefinitionListIndexedBase):
....
    class RuleSetModelInstanceRule(PartialRuleDefinition):
    ....
    def applicability_check(self, context, calc_vals, data):
        rmi_b = context.baseline
...
```

section21rule5 can be a good example for data processing at RMI level.

Reply:
Thank you.  I made an update.  If this is correct, I'll need to make the same changes to the other non-primary rules I created as part of this batch.